src/services/search/filter.js

```diff
@@ -40,10 +41,11 @@ export async function cleanFilter(search, mongo) {
 
   const result = search.filter((o) => {
     for (const match of dedupeMatches) 
-      if (o.href.includes(match) || o.title.includes(match)) 
+      if (o.href.toLowerCase().includes(match) || 
+        o.title.toLowerCase().includes(match) || 
+        o.snippet.toLowerCase().includes(match)) 
```

Comment:
jadi pengen nambahin https://eslint.org/docs/rules/operator-linebreak biar bisa gini, tapi nanti ae lah

```suggestion
      if (o.href.toLowerCase().includes(match)
          || o.title.toLowerCase().includes(match) 
          || o.snippet.toLowerCase().includes(match)) 
```